### DISCUSSION TITLE
Is it possible to create and test custom rules in ESLint Playground?

### DISCUSSION THREAD

**Discussion by yay** [eslint/eslint#16405] [Ideas]
That would be immensely helpful.

**Comment by kecrily** [comment]
It's not possible right now, but maybe we can consider implementing it

**Reply by yay** [reply]
@kecrily That would be great, thanks.

In the meantime, do you know what would be the best way to inspect the AST at runtime?
For example, I want to modify this plugin to allow limiting the number of variables destructured in the function signature:
https://github.com/lukeapage/eslint-plugin-destructuring/blob/master/src/rules/in-params.js

But I don't know anything about the AST structure, as this is JavaScript code and there are no type definitions either.

**Reply by aladdin-add** [reply]
you can try this: https://astexplorer.net/
